lib/LaTeXML/Util/Unicode.pm

```diff
@@ -326,6 +324,238 @@ sub unicode_mathvariant {
+  "\x{2A00}" => { role => 'SUMOP', need_scriptpos => 1, need_mathstyle => 1 },   # \bigodotmeaning=> ?
+"\x{2A01}" => { role => 'SUMOP', meaning => 'direct-sum', need_scriptpos => 1, need_mathstyle => 1 }, # \bigoplus
+"\x{2A02}" => { role => 'SUMOP', meaning => 'tensor-product', need_scriptpos => 1, need_mathstyle => 1 }, # \bigotimes
+"\x{2A04}" => { role => 'SUMOP', meaning => 'symmetric-difference', need_scriptpos => 1, need_mathstyle => 1 }, # \biguplus
+"\x{2A06}" => { role => 'SUMOP', meaning => 'square-union', need_scriptpos => 1, need_mathstyle => 1 }, # \bigsqcup
+  "\x{2A1D}" => { role => 'RELOP',      meaning => 'join' },                  # \Join
+  "\x{2AAF}" => { role => 'RELOP',      meaning => 'precedes-or-equals' },    # \preceq
+  "\x{2AB0}" => { role => 'RELOP',      meaning => 'succeeds-or-equals' },    # \succeq
+  "\x{FF0F}" => { role => 'OPFUNCTION', meaning => 'not' },                   # \not
+      #======================================================================
+  "arccos"  => { role => 'OPFUNCTION',   meaning => 'inverse-cosine' },          # \arccos #
+  "arcsin"  => { role => 'OPFUNCTION',   meaning => 'inverse-sine' },            # \arcsin #
+  "arctan"  => { role => 'OPFUNCTION',   meaning => 'inverse-tangent' },         # \arctan #
+  "arg"     => { role => 'OPFUNCTION',   meaning => 'argument' },                # \arg #
+  "cos"     => { role => 'TRIGFUNCTION', meaning => 'cosine' },                  # \cos #
+  "cosh"    => { role => 'TRIGFUNCTION', meaning => 'hyperbolic-cosine' },       # \cosh #
+  "cot"     => { role => 'TRIGFUNCTION', meaning => 'cotangent' },               # \cot #
+  "coth"    => { role => 'TRIGFUNCTION', meaning => 'hyperbolic-cotangent' },    # \coth #
+  "csc"     => { role => 'TRIGFUNCTION', meaning => 'cosecant' },                # \csc #
+  "deg"     => { role => 'OPFUNCTION',   meaning => 'degree' },                  # \deg #
+  "det"     => { role => 'LIMITOP',      meaning => 'determinant', need_scriptpos => 1 },    # \det #
+  "dim"     => { role => 'LIMITOP',      meaning => 'dimension' },                           # \dim #
+  "exp"     => { role => 'OPFUNCTION',   meaning => 'exponential' },                         # \exp #
+  "gcd"     => { role => 'OPFUNCTION',   meaning => 'gcd', need_scriptpos => 1 },            # \gcd #
+  "hom"     => { role => 'OPFUNCTION',   need_scriptpos => 1 },                              # \hom #
+  "inf"     => { role => 'LIMITOP',      meaning        => 'infimum', need_scriptpos => 1 }, # \inf #
+  "ker"     => { role => 'OPFUNCTION',   meaning        => 'kernel' },                       # \ker #
+  "lg"      => { role => 'OPFUNCTION' },                                                     # \lg #
+  "lim"     => { role => 'LIMITOP',    meaning => 'limit',          need_scriptpos => 1 }, # \lim #
+  "lim inf" => { role => 'LIMITOP',    meaning => 'limit-infimum',  need_scriptpos => 1 }, # \liminf #
```

Comment:
Interesting, first I assume single characters, then single words (no spaces), but I see some entries even have a space.

Reply:
yeah, these don't actually get fired, yet; it would need to be hooked into the math ligature that combines "s,i,n" into "sin"; and then looked up! I suspect "lim inf" will likely fail that test. I kinda left them all in as a reminder...